Code.Rmd

```diff
@@ -7,19 +7,30 @@ source("common.R")
 # R code {#r}
 
 The first principle of using a package is that all R code goes in `R/`.
-In this chapter, you'll learn about the `R/` directory, my recommendations for organising your functions into files, and some general tips on good style.
-You'll also learn about some important differences between functions in scripts and functions in packages.
+In this chapter, you'll learn about the `R/` directory, our recommendations for organising your functions into files, and some general tips on good style.
+You'll also learn some important differences between defining a function in a script and defining it in a package.
 
 ## R code workflow {#r-workflow}
 
-The first practical advantage to using a package is that it's easy to re-load your code.
-You can either run `devtools::load_all()`, or in RStudio press __Ctrl/Cmd + Shift + L__,  which also saves all open files, saving you a keystroke.
+As your use of R becomes more sophisticated, it's common to start to write your own R functions.
+If a function is only used in one place, you probably define it right there.
+But if you've bothered to write a function, it's likely you want to reuse it in multiple places: within one script, across multiple scripts, or even across multiple projects.
+
+The next logical step is to collect such function definitions in one or more dedicated `.R` files and then `source()` them as needed.
+Typically these functions co-evolve with the code where you use them, i.e. your analysis code, your Shiny apps, or your R Markdown reports.
+
+The first practical advantage to making your own functions available as a **package** is that it's easy to re-load them, as you refine their functionality or interface.
+Instead of re-running multiple `source()` commands, you can use `devtools::load_all()` to update all the function definitions in a package.
+
+:::rstudio-tip
+RStudio exposes `load_all()` in the *Build* menu, in the *Build* pane via *More > Load All*, and in keyboard shortcuts Ctrl + Shift + L (Windows & Linux) or Cmd + Shift + L (macOS). When run inside RStudio `load_all()` also saves all open files, saving you a keystroke.
+:::
 
 This keyboard shortcut leads to a fluid development workflow:
 
 1. Edit an R file.
 
-1. Press Ctrl/Cmd + Shift + L.
+1. Execute `load_all()`, probably via Ctrl/Cmd + Shift + L.
```

Comment:
Make it *slightly* less RStudio-specific.

Reply:
I'd rather keep the keyboard shortcut primary, because it is such an important part of the workflow.